wrapper/desktop/include/KHR/khrplatform.h

```diff
@@ -0,0 +1,285 @@
+#ifndef __khrplatform_h_
```

Comment:
yes, this code was just for trying some stuff out. 
I followed the tutorial [learnopengl.com](https://learnopengl.com/#!Getting-started/Hello-Window), there they say that one should add the code for the gad and this one into the include dir... 

As we move to the OpenMapperDesktop, this will be removed